modules/core/test/v2/unit/coins/hbar.ts

```diff
@@ -98,6 +99,21 @@ describe('Hedera Hashgraph:', function() {
     });
   });
 
+  describe('Sign Message', () => {
+    it('should be performed', async () => {
+      const keyPair = new accountLib.Hbar.KeyPair();
+      const messageToSign = Buffer.from(randomBytes(32)).toString('hex');
+      const signature = await basecoin.signMessage(keyPair.getKeys(), messageToSign);
+      keyPair.verifySignature(messageToSign, Uint8Array.from(Buffer.from(signature, 'hex'))).should.equals(true);
```

Comment:
It's not incorrect, but the extra `Uint8Array.from()` wrapper is unnecessary because `Buffer` is a subtype of `Uint8Array`:

```
$ node
> Buffer.from([]) instanceof Uint8Array
true
```